app/serializers/viewer_member_info_serializer.rb

```diff
@@ -9,11 +9,11 @@ class ViewerMemberInfoSerializer
 
   THUMB_DERIVATIVE = :thumb_mini
 
-  attr_reader :work, :current_user
+  attr_reader :work, :show_unpublished
 
-  def initialize(work, current_user:nil)
+  def initialize(work, show_unpublished: false)
     @work = work
-    @current_user = current_user
+    @show_unpublished = show_unpublished
   end
```

Comment:
I guess I can see how this is better, to just pass in the the boolean instead of the current_user.... until/unless this thing needs to determine something else baed on the current_user too, some more complicated permission....  I guess we can always pass in a second argument for that specific thing then. I think it's fine to pass in current_user, but also I can see the idea to just tell the presenter what it should do and not make it figure it out for itself... I dunno, ok, fine, what you chose!